js/src/settings/edit-contact-information/index.js

```diff
@@ -23,8 +23,11 @@ export default function EditContactInformation() {
 	const { updateGoogleMCContactInformation } = useAppDispatch();
 	const { data: address } = useStoreAddress();
 	const [ isSaving, setSaving ] = useState( false );
+
+	// The initial `isValid: true` is to avoid blocking the submission when only the store address is changed.
+	// And the prevention of saving invalid phone number can be checked by initial `isDirty: false`.
```

Comment:
❔ I'm confused by the code comment - prevent saving _invalid_ number by checking _isDirty_? Or do you actually mean the following? 🤔 

```suggestion
	// And the prevention of saving initial phone number can be checked by initial `isDirty: false`.
```

Note: Saving initial unchanged non-dirty phone number would override the phone verification status to become "unverified" in Google Merchant Center, even though the phone number was not changed at all, that's why we want to prevent it.